calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/entityproviders/CalendarEventEntityProvider.java

```diff
@@ -129,16 +138,23 @@ public boolean entityExists(final String id) {
 	 * Optional firstDate and lastDate query params in ISO-8601 date format, YYYY-MM-DD
 	 */
 	@EntityCustomAction(action = "my", viewKey = EntityView.VIEW_LIST)
-	public List<CalendarEventSummary> getMyCalendarEventsForAllSite(
+	@SuppressWarnings({"rawtypes", "unchecked"})
+	public List<?> getMyCalendarEventsForAllSite(
 			final EntityView view, final Map<String, Object> params) {
-		final List<CalendarEventSummary> rv = new ArrayList<CalendarEventSummary>();
+		
+		final List rv = new ArrayList<>();
 
 		// optional timerange
 		final TimeRange range = buildTimeRangeFromRequest(params);
+		
+		boolean detailed = false;
+		if (params.containsKey("detailed")) {
+			detailed = BooleanUtils.toBoolean((String) params.get("detailed"));
+		}
 
 		// add events form my workspace
 		final String siteId = "~" + this.developerHelperService.getCurrentUserId();
-		rv.addAll(getEventsForSite(siteId, range));
```

Comment:
Being fussy, you should really pull this from the SiteService in case the user site id schema ever changes (!)

SiteService.getUserSiteId(this.developerHelperService.getCurrentUserId());